src/makeRobot.js

```diff
@@ -39,6 +39,69 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name: name,
+    version: version,
+    wheels: wheels,
+    x: 0,
+    y: 0,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      return `name: ${
+        this.name}, chip version: ${this.version}, wheels: ${this.wheels}`;
+    },
+    get location() {
+      return `${this.name}: x=${this.x}, y=${this.y}`;
+    },
+    goForward(value = 1) {
+      if (value < 0) {
+        return this;
+      }
+      this.y += value;
+      this.coords.y = this.y;
+
+      return this;
+    },
+    goBack(value = 1) {
+      if (value < 0) {
+        return this;
+      }
+      this.y -= value;
+      this.coords.y = this.y;
```

Comment:
```suggestion
      if (value > 0) {
        this.y -= value;
      }
```
Зачем тебе в два места записывать?